src/torchcodec/decoders/_core/VideoDecoder.h

```diff
@@ -123,7 +123,14 @@ class VideoDecoder {
   // --------------------------------------------------------------------------
   // ADDING STREAMS API
   // --------------------------------------------------------------------------
-
+  enum ColorConversionLibrary {
+    // Let the implementation decide the color conversion library to use.
+    AUTO,
```

Comment:
Since AUTO is currently just FILTERGRAPH, I'd suggest to remove this and only add it back when we intend to change the default policy. It was a source of confusion to me (as in previous comments).

Reply:
Done.